app/models/notifier.js

```diff
@@ -15,6 +15,36 @@ export default Resource.extend({
     this._super(...args);
   },
 
+  displayNameAndType: computed('displayName', 'notifierType', function() {
+    const upperCaseType = (get(this, 'notifierType') || '').replace(/^\S/, (s) => {
+      return s.toUpperCase();
+    })
+
+    return `${ get(this, 'displayName') } (${ upperCaseType })`
+  }),
+
+  notifierTableLabel: computed('slackConfig', 'pagerdutyConfig', 'emailConfig', 'webhookConfig', function() {
+    const sc = this.get('slackConfig');
+    const pc = this.get('pagerdutyConfig');
+    const ec = this.get('smtpConfig');
+    const wc = this.get('webhookConfig');
+
+    if (sc) {
+      return 'Default Channel';
+    }
+    if (pc) {
+      return 'Service Key';
+    }
+    if (ec) {
+      return 'Default Recipient Address';
+    }
+    if (wc) {
+      return 'URL';
```

Comment:
Doesn't get checked multiple times but still seems like it belongs with where ever the other strings end up.